### PR TITLE
version bump v1.5.0

### PR DESCRIPTION
**Pull request by alon-e**
# Description
Bumping IRI version to v1.5.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,6 @@ It is specially designed for users seeking a fast, efficient and fully-compatibl
 
 Running an IRI node also allows light wallet users a node to directly connect to for their own wallet transactions.
 
-* **Latest release:** 1.4.2.4 Release
-* **License:** GPLv3
```
Comment (GalRogozinski) [inline README.md]: Why did you remove the license?

Comment (alon-e) [reply README.md]: re-added.
